lib/rules/web/admin_console/4.2/deployment.xyaml

```diff
@@ -88,3 +88,16 @@ check_current_cpu_utilization:
   element:
     selector:
       <<: *value
+check_pod_status:
+  elements:
+  - selector: &status
+      xpath: //tr/td[contains(.,'<pod_name>')]/following-sibling::td[2][contains(.,'<pod_status>')]
```

Comment:
I think we don't need to add index here, just to make sure the string is at the same row with pod name

Reply:
will update